tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -125,6 +136,16 @@ function renderGlobalModal(options?: Options) {
   return {...result, waitForModalToHide};
 }
 
+/**
+ * Helper that waits for the drawer to be hidden from the DOM. You may need to
+ * wait for the drawer to be removed to avoid any act warnings.
+ */
+function waitForDrawerToHide() {
+  return rtl.waitFor(() => {
+    expect(rtl.screen.queryByRole('complimentary')).not.toBeInTheDocument();
```

Comment:
Still think we should give the drawer some kind of label

```suggestion
    expect(rtl.screen.queryByRole('complimentary', {name: "Popout drawer"})).not.toBeInTheDocument();
```